lib/chef/provider/powershell_script.rb

```diff
@@ -24,71 +24,147 @@ class PowershellScript < Chef::Provider::WindowsScript
 
       provides :powershell_script, os: "windows"
 
+      def initialize (new_resource, run_context)
+        super(new_resource, run_context, '.ps1')
+        normalize_script_exit_status
```

Comment:
not to bike shed a name, but I we aren't doing anything with the exit status yet and that confused me. Something like "add_exit_status_wrapper" I wouldn't have questioned.